Makefile

```diff
@@ -189,9 +189,12 @@ $(BINDIR)/e2e.test: .init
 # Some prereq stuff
 ###################
 
-.init: $(scBuildImageTarget)
+.init: $(scBuildImageTarget) | $(BINDIR)
```

Comment:
neat. I learned something. This is an "order only prerequisite". In this context, it means to make the build directory if it doesn't exist, but just because it's timestamp is updated, doesn't mean it's invalidated the thing that depends on it.